### ISSUE TITLE
CORS configuration

### ISSUE THREAD

**Issue by KannarFr** [apache/pulsar-manager#219]
It should be nice to configure CORS allowed hosts from backend configuration in case of splitted frontend/backend runtimes and host.

**Comment by tuteng** [comment]
You are right. If front-end and back-end separation is used in the production environment, I suggest using load balance to configure it properly, such as nginx https://github.com/apache/pulsar-manager/tree/master/front-end#add-configuration-to-the-configuration-file-on-nginx-server.

**Comment by KannarFr** [comment]
Well in a cloud context I can't define a custom ngnix configuration and I think it should be more simple to just provide spring configuration as playframework does. Like 

```
play.filters.cors {
  pathPrefixes = ["/some/path", ...]
  allowedOrigins = ["http://www.example.com", ...]
  allowedHttpMethods = ["GET", "POST"]
  allowedHttpHeaders = ["Accept"]
  preflightMaxAge = 3 days
}
```
 
WDYT?

**Comment by tuteng** [comment]
I got it, I will consider adding this feature later.
